Makefile

```diff
@@ -88,6 +90,15 @@ $(STATUSGO): | deps
 	  $(MAKE) setup-dev && \
 	  $(MAKE) statusgo-library
 
+
+STATUSGOWINDOWS := vendor/status-go/build/bin/libstatus.dll.a
```

Comment:
Can't this be just "libstatus.a"?

Reply:
I'll change it to .lib since I found out that newer versions of mingw support linking against .lib files. I want this to have a different extension from .a to indicate that it is a file for windows (even if in the end, it becomes part of the resulting executable)